bogenliga/bogenliga-db-migration/src/main/resources/db/migration/LOCAL/V33__liga__insert_liga_data.sql

```diff
@@ -1,9 +1,9 @@
-INSERT INTO liga (liga_id, liga_region_id, liga_name, liga_uebergeordnet)
```

Comment:
Änderungen an bestehenden Flyway-Skripten ist in der Produktion nicht mehr möglich.
Das Flyway-Migrate würde scheitern.